### ISSUE TITLE
Error using web version of Bakta

### ISSUE THREAD

**Issue by corkdagga** [ag-computational-bio/bakta-web-ui#17]
Hi,

I am trying to annotate a genome using the web version of Bakta.

I am able to upload my fasta file but after I press the submit button, I get the following error:

500 - {"code":2,"message":"Get \"https://10.240.16.1:443/apis/argoproj.io/v1alpha1/namespaces/bakta/workflowtemplates/bakta-job-1.6.1\": EOF","details":[]} 

I have tried 2 different fasta files.

Just wondering, is this a problem with my files or some other problem?

Cheers

**Comment by St4NNi** [comment]
Hi,

Thanks for reporting this issue. We had some problems with the underlying infrastructure, these should be fixed by now. I will close this issue for now, feel free to comment / reopen this issue if the problem still exists.

BR

Sebastian

**Comment by AthinaGav** [comment]
Hi,

I just had the exact same error when I uploaded my fasta file:

500 - {"code":2,"message":"Get \"https://10.240.16.1:443/apis/argoproj.io/v1alpha1/namespaces/bakta/workflowtemplates/bakta-job-1.8.1\": EOF","details":[]} 

It actually was the reason why I tried to submit the same file again.
But on the Jobs tab, it looks like the jobs actually started: I see my file names under "Jobname" and "INIT" under "Status". But the "Last updated" column hasn't changed for 20 min now since the error, so I am not sure.

I am probably going to use the local version for my analysis, but I thought you would probably like to know that this issue persists.

Best wishes,
Athina
